### ISSUE TITLE
Enable Error prone during compile

### ISSUE THREAD

**Issue by bryanlb** [slackhq/kaldb#229]
http://errorprone.info/bugpattern/JavaDurationWithNanos

```java
Duration duration = Duration.ofSeconds(2);
Duration foo = duration.withNanos(10);
```

```bash
mvn compile
```

```bash
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  15.703 s
[INFO] Finished at: 2022-02-10T10:43:10-07:00
[INFO] ------------------------------------------------------------------------
```

No warnings thrown during compilation either.

**Comment by mansu** [comment]
Removed Errorprone as part of this [commit](https://github.com/slackhq/kaldb/commit/58531de6079fe8b7e99da3761148a152d55d019e). Updated the issue to add it back.

**Comment by danhermann** [comment]
Resolved by #383
